doc/build-osx.md

```diff
@@ -46,21 +46,19 @@ NOTE: Building with Qt4 is still supported, however, could result in a broken UI
 2.  Build bitcoind:
 
         ./autogen.sh
-        ./configure --with-gui=qt5
+        ./configure
         make
 
-3.  It is also a good idea to build and run the unit tests:
+    By default 'make' will also run the unit tests.
```

Comment:
Not sure why I thought this was the case. 'make' is still only compiling the tests...